### PR TITLE
Add docker to the web application

### PR DESCRIPTION
**Pull request by mansourisiwar**
Onboarding task: Add docker to the web application

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+FROM java:8-jdk-alpine
+
+# Add Maintainer Info
+LABEL maintainer="siwar.mansouri@vistaprint.com"
+
+# Add the application's jar to the container
+COPY ./build/libs/gs-spring-boot-0.1.0.jar /usr/app/
+
+WORKDIR /usr/app
+
```
Comment (ManelTahari) [inline Dockerfile:10]: You need to create a workdir before copying your jar into it.

**Comment by ManelTahari** [comment]
Our convention for naming branches is: DG-XXXXX-ticket's name
example: DG-27366-Add-docker-to-the-web-application
Our convention for naming commits is: DG-XXXXX: the change you're about to commit.
example: DG-27366: update readme file

**Comment by ManelTahari** [comment]
In the this post request it's better to change the output "Product is created successfully" by something more enlightening like the name of your product for example "Product with id: X was created successfully"
You'll find this at line 45 in:   
https://github.com/mansourisiwar/onBoarding/blob/master/src/main/java/com/example/demo/controller/ProductServiceController.java

Same goes for put and delete

**Comment by ManelTahari** [comment]
LGTM

**Review by ManelTahari — DISMISSED** [review]

**Comment by mbelhaouane** [comment]
I think that what @ManelTahari  wanted to say about the commits is that you have to go back to your previous commits and change their messages.

**Review by mbelhaouane — DISMISSED** [review]

**Pull request closed without merging**